services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -687,18 +687,18 @@ func (r *repository) afterTransform(ctx context.Context, transaction *sql.Tx, st
 	span, ctx := tracer.StartSpanFromContext(ctx, "afterTransform")
 	defer span.Finish()
 
-	configs, err := state.GetEnvironmentConfigs()
-	if err != nil {
-		return err
-	}
-	for env, config := range configs {
-		if config.ArgoCd != nil {
-			err := r.updateArgoCdApps(ctx, transaction, &state, env, config)
-			if err != nil {
-				return err
-			}
-		}
-	}
+	//configs, err := state.GetEnvironmentConfigs()
```

Comment:
please remove dead code

Reply:
but actually, I think this has to stay, otherwise the updateArgoCdApps is never called